.azure-pipelines.yml

```diff
@@ -0,0 +1,66 @@
+trigger:
+- master
+- releases/*
+
+jobs:
+  - job: Test_Linux
+    displayName: Run Tests on Linux
+    pool:
+      vmImage: "Ubuntu 16.04"
+    variables:
+      run_coveralls: true
+    strategy:
+      matrix:
+        Node_v10:
+          node_version: 10
+        Node_v8:
+          node_version: 8
+        Node_v6:
+          node_version: 6
+        Node_v4:
+          node_version: 4
+        Node_v0_12:
+          node_version: 0.12
+        Node_v0_10:
+          node_version: 0.10
+    steps:
+      - template: .azure-pipelines-steps.yml
+
+  - job: Test_Windows
+    displayName: Run Tests on Windows
+    pool:
+      vmImage: vs2017-win2016
+    strategy:
+      matrix:
+        Node_v10:
+          node_version: 10
+        Node_v8:
+          node_version: 8
+        Node_v6:
+          node_version: 6
+        Node_v4:
+          node_version: 4
+          npm_version: 2
+        Node_v0_12:
+          node_version: 0.12
+          npm_version: 2
+        Node_v0_10: 
+          node_version: 0.10
+          npm_version: 2
+    steps:
+      - template: .azure-pipelines-steps.yml
+
+  - job: Test_MacOS
+    displayName: Run Tests on MacOS
+    pool:
+      vmImage: macos-10.13
+    strategy:
+      matrix:
```

Comment:
Should we be testing all versions of node on Mac?  If it is fast enough, we probably should because I have a lot of Mac-specific fixes in vinyl-fs

Reply:
Yeah. Not sure why I wasn't just doing that to begin with. Updated appropriately

Reply:
Thanks!